common.blocks/select/_mode/select_mode_radio.bemhtml

```diff
@@ -1,12 +1,12 @@
 block('select').mod('mode', 'radio')(
-    def().match(this._checkedOptions)(function() {
+    def().match(function() { return this._checkedOptions; })(function() {
         var checkedOptions = this._checkedOptions,
             firstOption = this._firstOption;
         if(firstOption && !checkedOptions.length) {
             firstOption.checked = true;
             checkedOptions = [firstOption];
         }
-        applyNext({ _checkedOption : checkedOptions[0] });
+        return applyNext({ _checkedOption : checkedOptions[0] });
     }),
 
     content()(function() {
```

Comment:
So it was present in select.bemhtml? It is especially interesting to see, because this block appears to be present in the one of the desktop.pages, and I have verified that generated html is the same with old and new bem-xjst.